fixes/logr.go

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-package main
+package fixes
```

Comment:
I was trying to avoid spending a lot of logic on making this a generic framework for fixes.  I was aiming for "bespoke".  That said, most of this is reasonable cleanup.

If we're having "fixers" I would just as soon have an interface and put each fixer in its own pkg, so some of the method-call conversions are not needed...

Reply:
that's fine by be -- I mainly just kept this structure while refactoring to avoid too many changes at once.  Alternatively, just getting rid of the "general" fixes pattern is fine.

This is mainly a case of "I saw your TODO about getting rid of 'fixes', but wanted to tackle that in a follow up"